Application/EdFi.Ods.Common/Models/Resource/ModelComparers.cs

```diff
@@ -151,9 +151,9 @@ public bool Equals(T x, T y)
                     return false;
                 }
 
-                return string.Equals(x.PropertyName, y.PropertyName, StringComparison.InvariantCultureIgnoreCase)
+                return string.Equals(x.PropertyName, y.PropertyName, StringComparison.OrdinalIgnoreCase)
```

Comment:
Changed code to use `OrdinalIgnoreCase` as it is generally more efficient than `InvariantCultureIgnoreCase` (does slightly less work), and definitely meets our needs.